src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -2,10 +2,18 @@
 
 public class Task6 {
     public static double calculateS(double x) {
-        return 0d;
+        double s;
+        s = 1 + x;
+        s += (Math.pow(x,2))/2;
+        s+= (Math.pow(x,3))/6;
+        s+=(Math.pow(x,4))/24;
+
+        return s;
```

Comment:
Despite your solution is correct, try to solve this task using loop and factorial function(you need to write this function).

Case if you'll have task to calculate 10 more terms you'll have lots of boilerplate code.